org/mozilla/jss/CryptoManager.java

```diff
@@ -1224,11 +1249,21 @@ private native boolean verifyCertTempNative(byte[] certPackage,
      * have.
      * @return - The current ocsp policy in effect.
      */
-
     public static synchronized int getOCSPPolicy() {
         return ocspPolicy.ordinal();
     }
 
+    /**
```

Comment:
These should be pulled out into a separate commit. I guess it is fine to leave them in this PR.